src/app/(nextjs_migration)/(guest)/page.tsx

```diff
@@ -28,6 +29,7 @@ export default function Home() {
         crossOrigin="anonymous"
       />
       <Script type="module" src="/nextjs_migration/client/js/landing.js" />
+      <PageLoad pageTitle="Home" />
```

Comment:
This is just for stats logging and isn't user facing, correct? (if it is user facing, we'd need to localize)

Reply:
Also, below we have some code like a this:
```js
appEvents.buttonClick.record({ label: "sign-in" });
```

Should this be "home" to be consistent lowercasing w/ "sign-in"?

Reply:
I am going to change this `PageLoad` component to send the current URL path instead, so yes I agree with your comments but this is going to change.